person_class.rb

```diff
@@ -23,6 +25,16 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals << rental
+    rental.person = self
+  end
+
```

Comment:
- [ ] Here again, let's ensure the _person_ itself is sent as a parameter to create the new _rental_, thereby achieving the _'has-many'_ association. We can achieve this by modifying our code to look like the following example below:

```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
 ```

Reply:
This change has been done well